pages/dkp/kommander/1.3/projects/project-network-policies/index.md

```diff
@@ -48,39 +48,37 @@ To deny all egress traffic, select the **Egress** option here and then leave the
 ## Ingress Rules
 Ingress rules use a combination of **Port** / **Protocol** and **Source** to define the incoming traffic allowed to some or all of the pods in this namespace.
 
-The options under **Sources: From** allow you to define a source either by using the pod selector or by defining an IP block. When using the pod selector method, you can define the namespace, and/or the pods within that namespace. When you select a namespace in an ingress rule, you are allowing all of the pods in that namespace to receive incoming traffic that meets the other defined criteria.
+The options under **Sources: From** allow you to define a source either by using the pod selector or by defining an IP block. When using the pod selector method, you can define the namespace, and/or the pods within that namespace. 
 
-**Pods** - This option selects specific Pods, in the same namespace as the network policy, which should be allowed as ingress sources or egress destinations.
+**Namespaces** - When you select a namespace in an ingress rule source, you are allowing the pods selected by the pod selector in your selected namespaces to receive incoming traffic that meets the other defined criteria. If no pods have been selected, the rule will allow traffic from all pods in the selected namespaces.
 
-When you select both, you are selecting specific pods from a specific namespace.
+**Pods** - This option selects specific Pods which should be allowed as ingress sources or egress destinations. If no namespaces have been selected in the namespace selector, this will select all matching pods in the project namespace. Otherwise, it will select all matching pods in the selected namespaces.
 
 There also are options to select all namespaces and/or all pods.
 
-For example, if you wanted to create a default "deny all incoming traffic" policy for a namespace, you could select a namespace by name, and check the Select All Pods check box. By not defining any specific ingress rules, you explicitly create a "deny all traffic" network policy!
```

Comment:
This isn't totally accurate so I removed since we already covered above with `To deny all ingress traffic, select the **Ingress** option here and then leave the ingress rules empty`.